src/meta/src/hummock/manager/commit_epoch.rs

```diff
@@ -373,13 +318,18 @@ impl HummockManager {
             )?;
         }
 
-        let snapshot = HummockSnapshot {
-            committed_epoch: epoch,
-            current_epoch: epoch,
+        let snapshot = if is_visible_table_committed_epoch {
```

Comment:
what means `is_visible_table_committed_epoch` and what happens if a None snapshot is returned?

Reply:
As we are supporting partial checkpoint, we are deprecating the `max_committed_epoch` gradually. At current stage, the `max_committed_epoch` will have semantic as the `committed_epoch` of the state tables of the created streaming jobs. The state tables of the streaming job that are being created with the partial checkpoint will have different committed epoch during backfill. This `visible_table_committed_epoch` is also the `committed_epoch` of `HummockSnapshot`, so that the logic of frontend is not touched during the development.

When `Some` snapshot is returned, we will notify frontend about the snapshot update, and therefore when returning None, the frontend won't get notified.